src/main/kotlin/apply/ui/admin/mail/GroupMailTargetDialog.kt

```diff
@@ -75,6 +76,9 @@ class GroupMailTargetDialog(
             setItemLabelGenerator { it.toText() }
             addValueChangeListener {
                 mailTargetsGrid.setItems(mailTargetService.findMailTargets(evaluationItem.value.id, it.value))
+                if (it.value == EvaluationStatus.FAIL) {
+                    createNotification("점수가 0인 탈락자는 나오지 않습니다.", 3000)
```

Comment:
```suggestion
                    createNotification("점수가 0인 탈락자는 나오지 않습니다.")
```
혹쉬 duration을 따로 명시하신 이유가 있을까요??
다른 createNotification에서도 기본 duration 값을 사용하고 있으니 여기서도 기본 값을 써도 좋을 것 같아요 ㅎㅎ

Reply:
1초가 너무 짧더라고요. 잘 안 보여서 3초로 늘려주었어요.